src/reducers/highlightBullet.js

```diff
@@ -0,0 +1,7 @@
+export default (state, { value = false, draggedThoughtsRanked }) => {
+  return {
+    highlightBullet: value,
```

Comment:
Let's call this (and the file) `dragHold`. That shifts the semantics from the presentation to the behavior.